### ISSUE TITLE
[Table] Identify layout table

### ISSUE THREAD

**Issue by renow-luxembourg** [adobe/aem-core-wcm-components#2908]
## Feature Request
In French-speaking countries, for accessibility audit we use the RGAA accessibility guidelines.

**Is your feature request related to a problem? Please describe.**
In the criteria 5.3, we test if each layout table (`<table>` only used to display information in cells) has an attribute role="presentation" to success.
This test failed :
- if the table doesn't have the role="presentation"
- if the table has a caption (`<caption>` tags)
- if the table has headings (`<th>` tags) 
In some of our website, redactors use table to diplay content in column. For those websites, in the Accessibility Audit, we get a fail due to the missing role attribute.

**Describe the solution you'd like**
The simplest solution will be to automatically add the role="presentation" attribute to the `<table>` element when the user choose the "None" option in the Header selector.
On top of that a layout table shouldn't contain any `<caption>` tag, so it will be nice to disable this field when "None" is selected in the Header selector.

![Image](https://github.com/user-attachments/assets/77767e9c-ac14-4a13-9259-06cfa28eb067)

**Are there alternatives?**
We can also add a checkbox in the table tool to switch between data-table and layout-table.

**Documentation**

- W3C Tables tutorial : https://www.w3.org/WAI/tutorials/tables/
- W3C use of role=presentation : https://www.w3.org/WAI/ARIA/apg/practices/hiding-semantics/
- RAWeb 5.3.1 : https://accessibilite.public.lu/en/raweb1/criteres.html#test-5-3-1 (In english)
- RGAA 5.3.1 : https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#5.3.1 (In french)
- WCAG F49 : https://www.w3.org/WAI/WCAG21/Techniques/failures/F49
